tensorflow/compiler/xla/service/gpu/tests/reduction_vectorization_tile_fit_sm60.hlo

```diff
@@ -0,0 +1,20 @@
+// RUN: hlo_to_llvm_ir --ptx --sm=60 %s | FileCheck %s
```

Comment:
Can you please use the `// -----` separator to combine some of these tests into a single file, like in [here](https://github.com/tensorflow/tensorflow/blob/ddeb3ab137a3b8d8019582ee93ac3ed7f403cc26/tensorflow/compiler/xla/service/gpu/tests/scatter.hlo#L71)?

Reply:
I was looking for a way to reduce the number of file. Thanks fo the guidance.
It is done.